dali/base/dadfs.cpp

```diff
@@ -6271,20 +6271,22 @@ class CDistributedSuperFile: public CDistributedFileBase<IDistributedSuperFile>
     virtual IDistributedFile &querySubFile(unsigned idx,bool sub) override
     {
         CriticalBlock block (sect);
-        if (sub) {
+        if (sub)
+        {
```

Comment:
trivial: formatting change, now inconsistent with other existing code below.
This style is preferred, but if changing, need to make the rest of the function consistently use it.